src/tagmanager/tm_parser.c

```diff
@@ -1257,6 +1315,7 @@ static TMParserMap parser_map[] = {
 	MAP_ENTRY(TXT2TAGS),
 	MAP_ENTRY(ABC),
 	MAP_ENTRY(VERILOG),
+	MAP_ENTRY(SYSVERILOG),
```

Comment:
Move to the end, see below.

Reply:
Will do.
(Technically this one isn't really needed since the MAP_ENTRY() macro seems to take care of the order, but for consistency with the other two tm_parser\*.h files that *do* need it it'll be best to do it here as well).